client/apps/edit/components/content2/section_controls/index.styl

```diff
@@ -55,13 +55,23 @@
     .edit-controls
       max-width standard-overflow-w-margin
 
-.classic .edit-section__container
-  &[data-layout=column_width] .edit-controls
-    max-width classic-body-w-margin
-  &[data-layout=overflow_fillwidth],
-  &[data-type=image_set]
-    .edit-controls
-      max-width classic-overflow-w-margin
+.classic
```

Comment:
FWIW ideally this would be namespaced by component e.g. `.content2-classic` but probably not worth embarking on that refactor—just playing of-of-context CSS namespace cop.